apps/web/src/pages/templates/workflow/IntervalRadios.tsx

```diff
@@ -0,0 +1,49 @@
+import { Radio } from '@mantine/core';
+import { DigestUnitEnum } from '@novu/shared';
+import { Controller } from 'react-hook-form';
+import { useEnvController } from '../../../hooks';
+
+const options = [
+  { value: DigestUnitEnum.SECONDS, label: 'Sec' },
+  { value: DigestUnitEnum.MINUTES, label: 'Min' },
+  { value: DigestUnitEnum.HOURS, label: 'Hours' },
+  { value: DigestUnitEnum.DAYS, label: 'Days' },
```

Comment:
I guess it is a space and display issue but I find weird having some copies with abbreviations and others the full name.
Should I leave a comment about it in the Figma file?

Reply:
yes